src/generic/modal-iframe/ModalIframe.tsx

```diff
@@ -0,0 +1,44 @@
+import { forwardRef, ForwardedRef, IframeHTMLAttributes } from 'react';
+import classNames from 'classnames';
+
+interface ModalIframeProps extends IframeHTMLAttributes<HTMLIFrameElement> {
+  title: string;
+  className?: string;
+}
+
+const SANDBOX_OPTIONS = [
+  'allow-forms',
+  'allow-modals',
+  'allow-popups',
+  'allow-popups-to-escape-sandbox',
+  'allow-presentation',
+  'allow-same-origin',
+  'allow-scripts',
+  'allow-top-navigation-by-user-activation',
+].join(' ');
+
+export const IFRAME_FEATURE_POLICY = (
+  'microphone *; camera *; midi *; geolocation *; encrypted-media *, clipboard-write *'
```

Comment:
This has a typo - see https://github.com/openedx/frontend-app-learning/pull/1500

Also, the same code in frontend-app-learning has this important notice which seems like it should be here too:

https://github.com/openedx/frontend-app-learning/blob/0effb3231831ef54eb5951d3bfa85d4975989605/src/courseware/course/sequence/Unit/ContentIFrame.jsx#L12-L23